queue_services/entity-digital-credentials/requirements/bcregistry-libraries.txt

```diff
@@ -0,0 +1,3 @@
+git+https://github.com/bcgov/business-schemas.git@2.18.10#egg=registry_schemas
+git+https://github.com/seeker25/lear.git@16466___#egg=legal_api&subdirectory=legal-api
```

Comment:
Should this (and line 24 in requirements.txt) be forked to a bcgov repo so that we don't depend on a personal repo outside ministry control?

(Similar to what was done for [vue-test-utils-helpers](https://github.com/bcgov/business-filings-ui/blob/cb86797f7bd0923d58dc1af1950094bb4442bc58/package.json#L88)).